allennlp/semparse/contexts/sql_table_context.py

```diff
@@ -116,17 +111,23 @@ class SqlTableContext():
     def __init__(self,
                  all_tables: Dict[str, List[str]] = None,
                  tables_with_strings: Dict[str, List[str]] = None,
-                 database_directory: str = None) -> None:
+                 database_directory: str = None,
+                 utterances: List[str] = None) -> None:
+        self.grammar_dictionary = deepcopy(GRAMMAR_DICTIONARY)
+        self.utterances = utterances
```

Comment:
Why do you need the utterances here?  I thought this was supposed to be a general SQL class, not an ATIS one, and it doesn't look like you're actually using the utterances (or the tokenizer) inside this class.

Reply:
I'll remove it; I was trying to constrain dates in the `SqlTableContext` but it is no longer needed.